### DISCUSSION TITLE
How osu!lazer graphic output works for Windows 11?

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#30037] [Q&A]
I have 2 PCI-Ex videocards GTX970 (connected to display) and GTX 750 Ti (without SLI connection). It was interesting to test how osu will work on GTX 750 Ti, so I switched the game to OpenGL as Nvidia driver can calculate it on my second GPU. As a result I have 600fps, but the game feels much better. I even have no more accuracy problems on DT maps...
It looks like the answer in the fact that my both GPUs works at the same time for some reason. Both have 3D calculations: around 40% on 970 and 70% on 750Ti. Copy calculations is only on GTX750Ti. I can't understand what exactly doing my GTX 970, but it looks like with such parallel work performance is much better despite small fps.

**Comment by bytenekos** [comment]
Idk the exact configuration of your setup but I'll try to answer (this doesn't seem like an osu specific thing):

The frames are getting copied over from the 750ti (hence the copy)

If you're looking at gpu usage via the task manager, gpu load in there is dependent on gpu clock. So if the gpu clock is low (if the gpu is idling), it'll show a high 3d usage (at least that's how it is based on my observation)
